src/openapi/peridio-device-openapi.yaml

```diff
@@ -108,16 +108,16 @@ paths:
       description: |
         Returns information regarding whether an update is available or not.
 
-        If an update is available, additional information describing the update is returned including a presigned URL to acquire the update.
+        If no query string params are provided, the response will only contain the status.
```

Comment:
This change is incorrect ^^' it should be something like 

"By default we return only the status field, you can use the `expand` query parameter to obtain the other fields" 

Also check the other examples Daniel mentioned and try to explain them in this description c: